### PR TITLE
Resolves #2364

### LINKED ISSUE #2364: -Force and -NoClobber contradictory description
These 2 parameters are contradictory, NoClobber says Force will overwrite read-only file, Force says it won't do it.

`-Force` will overwrite an existing certificate file, **unless the Read-only or hidden attribute is set**
`-NoClobber` This parameter takes precedence over the Force parameter, **which permits this cmdlet to overwrite an existing certificate file, even if it has the Read-only attribute set**.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6c147fc5-e5c9-81ee-43b7-17638832572b
* Version Independent ID: aedf27ba-abfc-7e5d-4c69-cbf370fdac2b
* Content: [Export-Certificate (PKIClient)](https://docs.microsoft.com/en-us/powershell/module/pkiclient/export-certificate?view=windowsserver2019-ps)
* Content Source: [docset/winserver2019-ps/pkiclient/Export-Certificate.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/winserver2019-ps/pkiclient/Export-Certificate.md)
* Product: **w10**
* Technology: ****
* GitHub Login: @Kateyanne
* Microsoft Alias: **v-kaunu**

### PR DESCRIPTION
**Pull request by velkovb**
Fix parameter description
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2364

### CODE REVIEW

**Review by get-itips — APPROVED** [review]

File: docset/winserver2012-ps/pki/Export-Certificate.md [hunk 1]
```diff
@@ -119,7 +119,7 @@ Accept wildcard characters: False
 ```
 
 ### -Force
-Specifies that the exported certificate file will overwrite an existing certificate file, unless the Read-only or hidden attribute is set or the **NoClobber** parameter is also used.
+Specifies that the exported certificate file will overwrite an existing certificate file, even if it has the Read-only attribute is set.
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/pki/Export-Certificate.md:122]:
```suggestion
Specifies that the exported certificate file will overwrite an existing certificate file, even if it has the Read-only attribute set.
```

File: docset/winserver2012r2-ps/pki/Export-Certificate.md [hunk 2]
```diff
@@ -124,7 +124,7 @@ Accept wildcard characters: False
 ```
 
 ### -Force
-Specifies that the exported certificate file will overwrite an existing certificate file, unless the Read-only or hidden attribute is set or the **NoClobber** parameter is also used.
+Specifies that the exported certificate file will overwrite an existing certificate file, even if it has the Read-only attribute is set.
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/pki/Export-Certificate.md:127]:
```suggestion
Specifies that the exported certificate file will overwrite an existing certificate file, even if it has the Read-only attribute set.
```

File: docset/winserver2016-ps/pki/Export-Certificate.md [hunk 3]
```diff
@@ -121,7 +121,7 @@ Accept wildcard characters: False
 ```
 
 ### -Force
-Specifies that the exported certificate file will overwrite an existing certificate file, unless the Read-only or hidden attribute is set or the **NoClobber** parameter is also used.
+Specifies that the exported certificate file will overwrite an existing certificate file, even if it has the Read-only attribute is set.
```
Comment (JohanFreelancer9) [inline docset/winserver2016-ps/pki/Export-Certificate.md:124]:
```suggestion
Specifies that the exported certificate file will overwrite an existing certificate file, even if it has the Read-only attribute set.
```

File: docset/winserver2019-ps/pki/Export-Certificate.md [hunk 4]
```diff
@@ -121,7 +121,7 @@ Accept wildcard characters: False
 ```
 
 ### -Force
-Specifies that the exported certificate file will overwrite an existing certificate file, unless the Read-only or hidden attribute is set or the **NoClobber** parameter is also used.
+Specifies that the exported certificate file will overwrite an existing certificate file, even if it has the Read-only attribute is set.
```
Comment (JohanFreelancer9) [inline docset/winserver2019-ps/pki/Export-Certificate.md:124]:
```suggestion
Specifies that the exported certificate file will overwrite an existing certificate file, even if it has the Read-only attribute set.
```

File: docset/winserver2022-ps/pki/Export-Certificate.md [hunk 5]
```diff
@@ -121,7 +121,7 @@ Accept wildcard characters: False
 ```
 
 ### -Force
-Specifies that the exported certificate file will overwrite an existing certificate file, unless the Read-only or hidden attribute is set or the **NoClobber** parameter is also used.
+Specifies that the exported certificate file will overwrite an existing certificate file, even if it has the Read-only attribute is set.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/pki/Export-Certificate.md:124]:
```suggestion
Specifies that the exported certificate file will overwrite an existing certificate file, even if it has the Read-only attribute set.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by dariomws — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Pull request merged**